Formula/o/opentelemetry-cpp.rb

```diff
@@ -2,8 +2,8 @@ class OpentelemetryCpp < Formula
   desc "OpenTelemetry C++ Client"
   homepage "https://opentelemetry.io/"
   # TODO: Check if we can use unversioned `grpc` and `protobuf` at version bump.
```

Comment:
Was this checked?

Reply:
It should be possible to unversion at least `protobuf` (https://github.com/open-telemetry/opentelemetry-cpp/pull/2163).
Let's check if it'll work with `grpc` as well